static/src/stylesheets/module/facia/item-layouts/_fc-item--full-media-75.scss

```diff
@@ -101,4 +101,8 @@ Full item with 75% width media.
     .fc-item__avatar__media {
         right: -5%;
     }
+
+    .fc-item__liveblog-blocks {
+        height: 170px;
```

Comment:
what is 170? if it's lines of text, you can use the `get-line-height` sass function to get the height of multiples of particular font style's line-height.